Cargo.toml

```diff
@@ -22,6 +25,11 @@ imgui = "0.5"
 bytemuck = "1"
 smallvec = "1"
 
+# deps for simple_api_unstable
+winit = { version = "0.22", optional = true }
+futures = { version = "0.3", optional = true }
```

Comment:
Consider `pollster`, a small `block_on` lib with no deps. Besides that, you should depend directly on `futures-executor` to reduce deps.

Reply:
I'm open to trying this out and making it more minimal, when doing it I just used futures cause it was already used and compile time is not really a strong point for wgpu anyway but of course this does not mean we need to make it worse :) 

I kept using futures for the other examples/ did not change them to make them less surprising for now.